docs/source/whatsnew/v0.23.rst

```diff
@@ -0,0 +1,68 @@
+Version 0.23 (April 4, 2024)
+============================
+
+Cartopy has been relicensed from LGPL-3 to BSD-3-Clause. All contributions
+to Cartopy are now under the BSD-3-Clause license.
+
+Python 3.12 is now supported and the new minimum supported versions
```

Comment:
Should we mention numpy 2.0 in here too?

Reply:
Yes, good suggestion. Just added that note.